src/CodeGeneratorGo.cs

```diff
@@ -107,11 +107,35 @@ public override async Task Generate(CodeModel cm)
             // Version
             var versionTemplate = new VersionTemplate { Model = codeModel };
             await Write(versionTemplate, FormatFileName("version"));
+
+            // go.mod file, opt-in by specifying the gomod-root arg
+            var modRoot = Settings.Instance.Host.GetValue<string>("gomod-root").Result;

```

Comment:
the only thing it sucks here a little, that this needs to be added to all generation configs. and if authors screw it up what happens ? 
Should we have a stricter check on what the gomod-root can be ?

Reply:
Since the value is constant we can add this in the swagger-to-sdk config file in the repo so we don't have to specify in each config file.